src/NuGet.Clients/NuGetConsole.Host.PowerShell/PowerShellHost.cs

```diff
@@ -612,10 +602,8 @@ private void RestoreEvents_SolutionRestoreCompleted(SolutionRestoredEventArgs ar
 
         private bool ShouldNoOpDueToRestore(SolutionRestoredEventArgs latestRestore)
         {
-            return latestRestore != null &&
-                   _currentRestore != null &&
-                   latestRestore.SolutionSpecHash == _currentRestore.SolutionSpecHash &&
-                   latestRestore.IsSuccess == _currentRestore.IsSuccess;
+            return _currentRestore != null &&
```

Comment:
Deleted the previous comment.
Shouldn't this be something like if CurrentRestore is NoOp and latest restore was noOp or Succeeded?